metecho/api/model_mixins.py

```diff
@@ -162,3 +163,26 @@ def finalize_create_pr(self, *, error=None, originating_user_id):
             )
         else:
             self.notify_error(error=error, originating_user_id=originating_user_id)
+
+
+class SoftDeleteQuerySet(models.QuerySet):
+    def active(self):
+        return self.filter(deleted_at__isnull=True)
+
+    def delete(self):
+        return self.update(deleted_at=timezone.now())
```

Comment:
@wlonk This also needs to cascade, so that deleting a Project also deletes any Tasks and Orgs, and deleting a Task deletes any Orgs.

Reply:
Soft-deleting scratch orgs is in a different branch; should we merge #250 into this for ease of dev?

Reply:
@wlonk Sure

Reply:
Well, that got interesting, but it works!